src/04_printing.py

```diff
@@ -11,7 +11,10 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print("x is %s, y is %s, z is \"%s\"" % (x, y, z))
 
 # Use the 'format' string method to print the same thing
+print("x is {}, y is {}, z is \"{}\"".format(x, y, z))
 
-# Finally, print the same thing using an f-string
\ No newline at end of file
+# Finally, print the same thing using an f-string
+print(f"x is {x}, y is {y}, z is \"{z}\"")
```

Comment:
Did you check your precision? Make sure that you are careful on this matter! 🗡 
Try this: `print(f"x is {x:d}, y is {y:.3f}, z is {z}")`